FBTweak/_FBActionTweakTableViewCell.m

```diff
@@ -0,0 +1,67 @@
+/**
+ Copyright (c) 2014-present, Facebook, Inc.
+ All rights reserved.
+
+ This source code is licensed under the BSD-style license found in the
+ LICENSE file in the root directory of this source tree. An additional grant
+ of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import "FBTweak.h"
+#import "_FBActionTweakTableViewCell.h"
+
+@implementation _FBActionTweakTableViewCell
+
+@synthesize tweak = _tweak;
+
+/// Block to execute when a key path value has changed. The block provides the \c cell to update and
+/// the observed \v value that should be updated in \c cell. The block is always called on the main
+/// thread.
+typedef void (^FBActionTweakTableViewCellBlock)(_FBActionTweakTableViewCell *cell, id value);
+
+/// Returns a map of keys path to observe changes, to key paths to bind them.
```

Comment:
Hard to understand, please rewrite.

Reply:
Removed. It's easier to understand via code.